front/components/AppLayout.tsx

```diff
@@ -95,14 +95,26 @@ export default function AppLayout({
                             onWorkspaceUpdate={(workspace) => {
                               if (workspace.id === owner.id) {
                                 if (dataSource) {
-                                  router.push(`/w/${workspace.sId}/ds`);
+                                  router
+                                    .push(`/w/${workspace.sId}/ds`)
+                                    .then((e) => console.log(e))
```

Comment:
do we really want to `.then` ? `.catch` is enough here no ?

Reply:
It was a mistake. Thanks for pointing it out. fixed.